torchvision/models/detection/retinanet.py

```diff
@@ -5,7 +5,7 @@
 import torch
 import torch.nn as nn
 from torch import Tensor
-from torch.jit.annotations import Dict, List, Tuple
+from torch.jit.annotations import Dict, List, Tuple, Optional
```

Comment:
nit: is this needed?

Reply:
It is used in here

https://github.com/pytorch/vision/blob/8e244797efb64b2761fd61855fcac0123c10b804/torchvision/models/detection/retinanet.py#L459

Reply:
Ok. Let's keep this here then, as when we move the `# type:` annotations to be inline the import will be necessary